infer/src/absint/ProcCfg.ml

```diff
@@ -201,6 +201,100 @@ module Normal = struct
   let wto = Procdesc.get_wto
 end
 
+(** Forward CFG with exceptional control-flow for throw exception instruction only*)
+module ExceptionalThrowOnly = struct
+  module Node = DefaultNode
```

Comment:
Can we use `include Exceptional` to avoid the duplicated definitions? We can remove most of them except `fold_normal_or_exn_succs`, `fold_succs`, and `fold_preds` that have different definitions.

Reply:
Done. Thanks for suggestion!